haskell-language-server.cabal

```diff
@@ -158,7 +162,7 @@ flag cabalgild
   manual:      True
 
 common cabalgild
-  if flag(cabalgild)
+  if !flag(cabalgild)
```

Comment:
This looks undesirable?

Reply:
Also nitpick: I see 4x `flag(` and 56 x `flag (` (with space after flag.
I'd unify those for easier searchability..

Reply:
OOps thanks, I fixed that locally but forgot to push!